backend/src/app/rpc/mutations/teams.clj

```diff
@@ -330,12 +330,12 @@
 
 (declare create-team-invitation)
 
-(s/def ::email ::us/email)
+(s/def ::emails ::us/set-of-emails)
 (s/def ::invite-team-member
-  (s/keys :req-un [::profile-id ::team-id ::email ::role]))
+  (s/keys :req-un [::profile-id ::team-id ::emails ::role]))
 
 (sv/defmethod ::invite-team-member
-  [{:keys [pool] :as cfg} {:keys [profile-id team-id email role] :as params}]
+  [{:keys [pool] :as cfg} {:keys [profile-id team-id emails role] :as params}]
```

Comment:
we should keep the both props or reuse it conditionally (i mean email can be single item or set of items).
This is for backward compatibility. We can put a TODO note to remove the "backward compatibility code" for the next release where are way to have incompatible version of frontend running.